question-service/src/controller/question_service_handlers/delete_question_handler.ts

```diff
@@ -9,17 +9,25 @@ class DeleteQuestionHandler implements IApiHandler<DeleteQuestionRequest, Delete
     this.questionStore = storage.getQuestionStore();
   }
 
-  async handle(request: DeleteQuestionRequest): Promise<DeleteQuestionResponse> {
+  async handle(apiRequest: ApiRequest<DeleteQuestionRequest>):
+  Promise<ApiResponse<DeleteQuestionResponse>> {
+    const { request } = apiRequest;
     if (request.questionId <= 0) {
       return {
```

Comment:
Minor nit: This can be a `buildErrorResponse()` like the other handlers

Reply:
This API only returns an error message and no other field, so there is not really an `ErrorResponse` per se. Both an error and errorless responses have the exact same structure with varying message, so I did not see the point in abstracting it out.